### PR TITLE
EP-136: Publish package to JFrog Artifactory

### PR DESCRIPTION
**Pull request by aslotte**
## JIRA Ticket

- [ ] PR Title should be in the form: "DE-12345: Short Description of Changes"

## General

- [ ] Not applicable
- [ ] Changelog is added in README.md
  - [ ] Changes are at a high level and do not reference code directly
  - [ ] Scanned for spelling errors (eg. "Code Spell Checker" in VS Code)

### Testing Completed

- [ ] Tested manually locally
- [ ] Unit tests updated
- [ ] Integration tests updated
- [ ] None (please explain)

## Automated testing

- [ ] Unit tests
- [ ] Integration tests
- [ ] API tests
- [ ] End-To-End tests
- [ ] N/A (please clarify)

### CODE REVIEW

**Review by mlipschultz — APPROVED** [review]

**Comment by aslotte** [comment]
Thanks for the ✅ @mlipschultz! :) Let's not merge this one just yet though as it's a PoC and we'll probably at least change the secret names (e.g. moving them to org secrets)

File: README.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ Utility functions for Tetra Task Scripts
 - [Datetime Parser](#datetime-parser)
 - [Test](#test)
```
Comment (mlipschultz) [inline README.md:10]: This section still references `pipenv`, even though the project has migrated to `poetry`.

**Review by mlipschultz — CHANGES_REQUESTED** [review]
The README file needs to be updated

File: README.md (hunk 1 above)
Comment (VladimirShteyn) [reply README.md:10]: Fixed by deleting the `## Test` section. Environment setup and automated testing is in CONTRIBUTING.md.

File: pyproject.toml [hunk 2]
```diff
@@ -10,3 +10,35 @@ line-length = 88
 target-version = ['py37', 'py38', 'py39']
 include = '\.pyi?$'
 extend-exclude = 'snapshots'
+
+[tool.poetry]
+name = "ts-task-script-utils"
+version = "1.1.2"
```
Comment (mlipschultz) [inline pyproject.toml:16]: [We're now at 1.2.0](https://github.com/tetrascience/ts-task-script-utils/blob/master/pyproject.toml#L9)

**Review by mlipschultz — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (VladimirShteyn) [reply pyproject.toml:16]: -  [ ] bump package version by running `poetry version 1.2.0`

**Pull request closed without merging**